recipes/thorvg/all/conanfile.py

```diff
@@ -129,6 +129,8 @@ def requirements(self):
         if self.settings.os == "Linux":
             if self.options.with_engines in ["gl", "gl_beta"]:
                 self.requires("opengl/system")
+        if self.options.with_threads and Version(self.version) >= "0.15.0" and self.settings.compiler in ["clang", "apple-clang"]:
+            self.requires("llvm-openmp/17.0.4", transitive_headers=True, transitive_libs=True)
```

Comment:
Do we have logs showing any failure if any of these or even both parameters are removed from here?

```suggestion
            self.requires("llvm-openmp/17.0.4")
```